packages/build-tools/src/steps/functionGroups/build.ts

```diff
@@ -130,7 +132,7 @@ function createStepsForIosBuildWithCredentials({
     configureIosCredentialsFunction().createBuildStepFromFunctionCall(globalCtx),
     configureIosVersionFunction().createBuildStepFromFunctionCall(globalCtx),
     generateGymfile,
-    runFastlaneFunction().createBuildStepFromFunctionCall(globalCtx),
+    runFastlane.createBuildStepFromFunctionCall(globalCtx, { id: runFastlane.id }),
```

Comment:
What will be the `runFastlane.id` resolved to? If it's some nondeterministic random UUID I suggest making it const so users can reuse it.
```suggestion
    runFastlaneFunction().createBuildStepFromFunctionCall(globalCtx, { id: 'run_fastlane' }),
```

Reply:
That's exactly the reason for this change - it was a nondeterministic random UUID, now it is the same as the id defined in the `steps/functions/runFastlane.ts`, which is `run_fastlane`